src/main/java/com/keboola/tableexporter/Application.java

```diff
@@ -74,8 +77,10 @@ private static void connectDb() throws ApplicationException, UserException {
         connectionProps.put("password", dbPassword);
         connectionProps.put("useFetchSizeWithLongColumn", "true");
         connectionProps.put("defaultRowPrefetch", "50");
+
+        System.setProperty("oracle.net.tns_admin", tnsnamesPath);
         try {
-            connectionString.append("jdbc:oracle:thin:@").append(dbHost).append(":").append(dbPort).append(":").append(dbName);
+            connectionString.append("jdbc:oracle:thin:@").append(tnsnamesService);

```

Comment:
Nejak sa mi to nezda.

Povodny kod:
https://github.com/keboola/java-oracle-exporter/blob/bec531ced7d5d71bc9e00726982f88101f547c28/src/main/java/com/keboola/tableexporter/Application.java#L77-L90

**V povodnom kode:**
- sa to najprv pripajalo na `.append(dbPort).append(":").append(dbName);`
- ak to nevyslo tak na `.append(dbPort).append("/").append(dbName)`

**Teraz:**
- sa pripajas cez `tnsnames` ,. ... ~sa to najprv pripajalo na `.append(dbPort).append(":").append(dbName);`~
- ak to nevyslo tak na `.append(dbPort).append("/").append(dbName)`

Nechces tam tu povodnu funkcionalitu zachovat? Preco? Mozes mi to prosim podrobnesie vysvetlit?

Cakal by som, ze sa `tnsnames` prida ako prvy krok, a ostatne ostanu ... alebo ze sa to uplne nahradi.

Okrem toho nesedia logy, pripajas sa cez `tnsnames` ale logujes:
```
Connecting user " + dbUser + " to database " + dbName + " at " + dbHost + " on port " + dbPort
```

Tj. tie parametre za iba zaloguju, ale co ti zarucuje ze zodpovedaju `tnsnames`?
Logovat by sa malo teda to `tnsnames`, a ostatne veci mozes zalogovat napr. v PHP ked generujes to `tnsnames`.

-------------- 

Teda v skratke v tom treba urobit poriadok, rozhodnut sa:
- ci sa ma podporovat iba `tnsnames` a teda ostatne parametre vyhodit prec
- alebo podporovat oboje, bud ako alternativu (bud sa zada `tnsnames` alebo standardne parametere), alebo ako fallback (budu tie parametre s tnsnames dostupne? budu v configu?)

Reply:
ty voe, jsem byl nějakej mimo :-D no jasně, chtěl jsem to udělat tak že exporter bude podporovat jen tnsnames a v oracle extractoru by se skládal....ještě jednou sry tohle jhsem vůbec neměl dávat na review :D

Reply:
v pohode :) tazky bol ten minuly tyzden, tiez som bol v piatok uplne mimo

Reply:
jinak myšlenka je taková že java-oracle-exporter bude podporovat jen tnsnames který bude skládat oracle-extractor.... je to z toho důvodu že do tnsnames jde zadat více nastavení např. až se nám podaří to sslko tak protocol se nahradí `tcp` -> `tcps` a musel by se ten tnsnames stejně generovat - proto je imho lepší když ten javovský exporter bude podporovat jen tnsnames

Reply:
ok, to dava zmysel.

Reply:
koukni na to prosím tě ještě jednou při pondělku :)
- je tam podpora pouze tnsnames
- logování tam je pouze na uživatele + na servisu, která se použije ze souboru tnsnames.ora
- upravil jsem readme pro spouštění příkazů